### PR TITLE
Feat/self serve branding

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

This PR adds new pages to simplify and extend the branding process for the end user.  It adds the following screens:

- View branding settings
- Choose GOC branding
- Choose from pool of org branding
- Preview branding
- Request new branding

## Feature workflow
image

## How to access branding settings
- From your **service dashboard**, click **Settings**
- In the **Email** section, click **Change** on the **Email branding** row

## Artefacts
- [Miro board](https://miro.com/app/board/uXjVNqNAmeE=/)
- [Figma screens](https://www.figma.com/file/a1TQZWGPRijQkAfa7DHWXF/Service-branding-%7C-Image-de-service?type=design&node-id=205-403&mode=design&t=BC9cIUNLAnRPeMhb-0)

## Related Issues | Cartes liées
- https://github.com/cds-snc/notification-planning/issues/1157

# Test instructions | Instructions pour tester la modification

## Feature flag
- [x] Feature flag **ON** allows access to new branding pages
- [x] Feature flag **OFF** shows old branding pages

## Other
- [x] Branding can be seen previewed from settings
- [x] GOC branding can be selected
- [x] Branding from org pool can be selected
- [x] New branding can be requested
- [x] Admins can apply new branding to existing services

# Release Instructions | Instructions pour le déploiement

None.

# Reviewer checklist | Liste de vérification du réviseur

- [x] This PR does not break existing functionality.
- [x] This PR does not violate GCNotify's privacy policies.
- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
- [x] This PR does not significantly alter performance.
- [x] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -661,7 +661,7 @@ def useful_headers_after_request(response):
             "object-src 'self';"
             f"style-src 'self' fonts.googleapis.com https://tagmanager.google.com https://fonts.googleapis.com 'unsafe-inline';"
             f"font-src 'self' {asset_domain} fonts.googleapis.com fonts.gstatic.com *.gstatic.com data:;"
-            f"img-src 'self' {asset_domain} *.canada.ca *.cdssandbox.xyz *.google-analytics.com *.googletagmanager.com *.notifications.service.gov.uk *.gstatic.com https://siteintercept.qualtrics.com data:;"  # noqa: E501
+            f"img-src 'self' blob: {asset_domain} *.canada.ca *.cdssandbox.xyz *.google-analytics.com *.googletagmanager.com *.notifications.service.gov.uk *.gstatic.com https://siteintercept.qualtrics.com data:;"  # noqa: E501
             "frame-ancestors 'self';"
```
Comment (andrewleith) [inline app/__init__.py:665]: This change is to make the image preview work within the branding request page.  I ran this updated CSP through several CSP validators and none of them flagged it so I think this is a safe change.  The technique for loading images like this was taken from the [MDN example here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#examples)

**Review by jzbahrai — APPROVED** [review]
went through this! Based on our offline code review comments + 1 comment, I don't have anything to add. good job on the PR. There is a lot of french translation, I am hoping nothing was missed during testing/ but we can ask people to concentrate on it during the bug bash as well.

**Review by whabanks — APPROVED** [review]

File: app/main/forms.py [hunk 2]
```diff
@@ -1757,3 +1757,38 @@ class GoLiveAboutNotificationsFormNoOrg(GoLiveAboutServiceFormNoOrg):
         ],
         validators=[DataRequired()],
     )
+
+
+class BrandingGOCForm(StripWhitespaceForm):
+    goc_branding = RadioField(
```
[the remark this answers is not in the document]
Comment (andrewleith) [reply app/main/forms.py:1770]: Done!

File: app/main/views/service_settings.py [hunk 3]
```diff
@@ -1290,6 +1290,18 @@ def link_service_to_organisation(service_id):
 @user_has_permissions("manage_service")
 def view_branding_settings(service_id):
     def _get_current_branding():
+        """
```
Comment (jzbahrai) [inline app/main/views/service_settings.py:1293]: thank you for adding this!

**Pull request merged**